mimic/rest/ironic_api.py

```diff
@@ -21,6 +21,20 @@ def __init__(self, core):
         """
         self.core = core
 
+    @app.route('/nodes', methods=['POST'])
+    def create_node(self, request):
+        """
+        Responds with response code 201 and returns the newly created node.
+        """
+        return self.core.ironic_node_store.create_node(request)
+
+    @app.route('/nodes/<string:node_id>', methods=['DELETE'])
+    def delete_node(self, request, node_id):
+        """
+        Responds with response code 204 and delete the node.
+        """
+        return self.core.ironic_node_store.delete_node(request, node_id)
```

Comment:
I agree and thought about commenting on this in the last Ironic review... however that seems to be how mimic is organized already, so this seems fine to me for now. (But we should totally refactor that one day :)